ccd-definition/CaseEvent/CareSupervision/DEPRECATED.json

```diff
@@ -366,5 +366,22 @@
     "ShowSummary": "N",
     "ShowEventNotes": "Y",
     "EndButtonLabel": "Submit"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "sendOrderReminder",
```

Comment:
I think whilst we're deprecating it, we should remove all permissions in MultiState.json to stop anyone using it